runtime/queries/bitbake/indents.scm

```diff
@@ -1,101 +1,144 @@
+
 ((while_statement) @indent.begin
- (#set! indent.immediate 1))
+  (#set! indent.immediate 1))
+
 ((try_statement) @indent.begin
- (#set! indent.immediate 1))
-(ERROR "try" ":" @indent.begin (#set! indent.immediate 1))
+  (#set! indent.immediate 1))
+
+(ERROR
+  "try"
+  ":" @indent.begin
+  (#set! indent.immediate 1))
+
 ((python_function_definition) @indent.begin
- (#set! indent.immediate 1))
+  (#set! indent.immediate 1))
+
 (function_definition) @indent.begin
+
 (anonymous_python_function) @indent.begin
+
 ((with_statement) @indent.begin
- (#set! indent.immediate 1))
+  (#set! indent.immediate 1))
 
 (if_statement
   condition: (parenthesized_expression) @indent.align
   (#set! indent.open_delimiter "(")
   (#set! indent.close_delimiter ")")
   (#set! indent.avoid_last_matching_next 1))
+
 (while_statement
   condition: (parenthesized_expression) @indent.align
   (#set! indent.open_delimiter "(")
   (#set! indent.close_delimiter ")")
   (#set! indent.avoid_last_matching_next 1))
 
-(ERROR "(" @indent.align (#set! indent.open_delimiter "(") (#set! indent.close_delimiter ")") . (_)) 
+(ERROR
+  "(" @indent.align
+  (#set! indent.open_delimiter "(")
+  (#set! indent.close_delimiter ")")
+  .
+  (_))
+
 ((argument_list) @indent.align
- (#set! indent.open_delimiter "(")
- (#set! indent.close_delimiter ")"))
+  (#set! indent.open_delimiter "(")
+  (#set! indent.close_delimiter ")"))
+
 ((parameters) @indent.align
- (#set! indent.open_delimiter "(")
- (#set! indent.close_delimiter ")")
- (#set! indent.avoid_last_matching_next 1))
+  (#set! indent.open_delimiter "(")
+  (#set! indent.close_delimiter ")")
+  (#set! indent.avoid_last_matching_next 1))
+
 ((tuple) @indent.align
- (#set! indent.open_delimiter "(")
- (#set! indent.close_delimiter ")"))
+  (#set! indent.open_delimiter "(")
+  (#set! indent.close_delimiter ")"))
 
-(ERROR "[" @indent.align (#set! indent.open_delimiter "[") (#set! indent.close_delimiter "]") . (_)) 
+(ERROR
+  "[" @indent.align
+  (#set! indent.open_delimiter "[")
+  (#set! indent.close_delimiter "]")
+  .
+  (_))
 
-(ERROR "{" @indent.align (#set! indent.open_delimiter "{") (#set! indent.close_delimiter "}") . (_)) 
+(ERROR
+  "{" @indent.align
+  (#set! indent.open_delimiter "{")
+  (#set! indent.close_delimiter "}")
+  .
+  (_))
 
 [
   (break_statement)
   (continue_statement)
 ] @indent.dedent
 
 (ERROR
-  (_) @indent.branch ":" .
+  (_) @indent.branch
+  ":"
+  .
   (#lua-match? @indent.branch "^else"))
 
 (ERROR
-  (_) @indent.branch @indent.dedent ":" .
+  (_) @indent.branch @indent.dedent
+  ":"
+  .
   (#lua-match? @indent.branch "^elif"))
 
-(parenthesized_expression ")" @indent.end)
-(generator_expression ")" @indent.end)
-(list_comprehension "]" @indent.end)
-(set_comprehension "}" @indent.end)
-(dictionary_comprehension "}" @indent.end)
+(parenthesized_expression
+  ")" @indent.end)
+
+(generator_expression
+  ")" @indent.end)
+
+(list_comprehension
+  "]" @indent.end)
+
+(set_comprehension
```

Comment:
Can encapsulated literal captures like these remain in a single line?

Reply:
Resolved :+1:

Reply:
I think we should be consistent here -- either _all_ "simple" patterns are collapsed or none. I don't think it makes a difference whether the node is anonymous are not, as far as formatting is concerned.

(I have a slight preference for keeping the current pattern so the nesting is consistent across _all_ patterns, no matter how simple, but don't feel strongly enough to fight about it.)

Reply:
I only added back the inlining for anonymous captures without preds/directives, as they cannot be deeper than 1 layer.

For named nodes, lists, groupings, fields and such, they are all kept as before the changes.

Also I do believe it can be disruptive so it's currently put into a separate commit for easier review.

Reply:
> I only added back the inlining for anonymous captures without preds/directives/anchors/quantifiers, as they cannot be deeper than 1 layer.

I understand that those "simple" captures (without etc.) look nicer collapsed; I'm just saying if you do it for anonymous nodes, you should also do it for named nodes -- i.e., no special case for anonymous nodes.

Reply:
It's just a bit too hard to cover for multiple depths, and another is "what is considered as a simple capture". I can only reliably support anonymous nodes as they are defined depths

Reply:
Then I am against this change, I'm afraid.

Reply:
Some issues related to this:
anchors (`.`) only considers named nodes, and doesn't support `(foo . .)`. So there's zero way to declare if there's no child here. I don't think having a multi-depth queries will be nice